src/test/java/com/tikal/hudson/plugins/notification/ProtocolTest.java

```diff
@@ -170,9 +171,15 @@ private UrlFactory startSecureServer(Servlet servlet, String path, String author
     ServerConnector connector = new ServerConnector(server);
     server.setConnectors(new Connector[] {connector});
 
-    ServletHandler servletHandler = new ServletHandler();
-    server.setHandler(servletHandler);
-    servletHandler.addServletWithMapping(new ServletHolder(servlet), path);
+    // Create a ServletContextHandler to handle servlets
```

Comment:
These comments just restate what is being done in the code. Such comments don't help the reader very much and are likely to become out of date, so I would just delete them. The best code comments explain the reason why a particular thing is being done rather than describing what is being done.

Reply:
Thank you @basil for the feedback.